gnosis/safe-react
# Bug: Add custom app feature works weird or doesn't work

**What**:
Related to #2221. There is a problem reported where sometimes the input does not trigger the request for retrieving the sapp manifest.



**Why**:



**How**:
I wasn't able to find a way to reproduce it, I've talked with Ivan and we agreed on adding a loading indicator and try to gather more feedback on when the problem arises. 
**This is NOT a fix for the problem reported**



**How to test it**:
Add a safe-app URL, for example, this one https://rimeissner.dev/sapp-recorder/ and check that while the sapp is being loaded, the Input label will show the text "Loading...".
